deps/init.js

```diff
@@ -0,0 +1,18 @@
+db = connect("mongodb://mongodb:27017/orchestrator");
```

Comment:
Add a BIG comment explaining this. This is a ugly workaround we need to fix
(related to SNOS not able to prove first block)

Reply:
Change name for:
- ugly_workorund_snos_first_block xD (or something like that)